docs: Add Msft SQL Server client examples and link in sql server guide (#23263)
gravitational/teleport: docs/pages/connect-your-client/gui-clients.mdx

![DynamoDB local connection in NoSQL Workbench](../../img/database-access/nosql-workbench-connection.png)

## SQL Server with Azure Data Studio

In Azure Data Studio create a connection using your proxy's endpoint. This is

> `localhost,62652` in the example above. On a Windows machine, using an address in the format `127.0.0.1,62652` could be required instead of `localhost`. (See

Author: Done

[Get connection information](./gui-clients.mdx#get-connection-information) for
more information.)

Create a connection with Microsoft SQL Server with these settings:

|Connection Detail|Value|
|---|---|
|Server|`host`,`port` of proxy endpoint|
|Authentication Type|SQL Login|
|Password|empty|
|Encrypt|`False`|

Example:
![Azure Data Studio connection options](../../img/database-access/azure-data-studio-connection.png)

> Click Connect to connect.

Reviewer: This sentence seems redundant and/or obvious. I suggest removing it. If not, stylize the reference to the button:

```suggestion
Click **Connect** to connect.
```

Click "Save", and pgAdmin should immediately connect. If pgAdmin prompts you
for password, leave the password field empty and click OK.

## Microsoft SQL Server Management Studio

In Microsoft SQL Server Management Studio connect to a database engine using your proxy's endpoint. This is

> `localhost,62652` in the example above. Using the IP `127.0.0.1,62652` connection could be required on

Author: please see updates @alexfornuto

![DynamoDB local connection in NoSQL Workbench](../../img/database-access/nosql-workbench-connection.png)

> ## SQL Server Azure Data Studio

Reviewer 2: This is actually two products but it sounds like one. 
```suggestion
## SQL Server with Azure Data Studio
```

![DynamoDB local connection in NoSQL Workbench](../../img/database-access/nosql-workbench-connection.png)

## SQL Server Azure Data Studio

In Azure Data Studio create a connection using your proxy's endpoint. This is

> `localhost,62652` in the example above. Using the IP `127.0.0.1,62652` host could be required on
> a Windows machine instead of `localhost`. (See
> [Get connection information](./gui-clients.mdx#get-connection-information) for
> more information.)

Reviewer 3: ```suggestion
`localhost,62652` in the example above. On a Windows machine, using an address in the format `127.0.0.1,62652` could be required instead of `localhost`. (See
[Get connection information](./gui-clients.mdx#get-connection-information) for
more information.)
```

Reviewer 3: Does the `tsh proxy db` command indicate whether you can use `localhost` or not? If that's the case, maybe we can say so explicitly instead of linking to the "Get connection information" section.

> Create a connection with Microsoft SQL Server with the proxy endpoint `host`,`port`

Reviewer 3: This paragraph is a bit jumbled to me. Would it be possible to use a bulleted list or table to indicate which values to use for which fields?

Click "OK" to connect.

## SQL Server Microsoft SQL Server Management Studio

In Microsoft SQL Server Management Studio connect to a database engine using your proxy's endpoint. This is

> `localhost,62652` in the example above. Using the IP `127.0.0.1,62652` connection could be required on

Reviewer 3: Same comment as the similar paragraph for Azure Data Studio

a Windows machine instead of `localhost`. (See
[Get connection information](./gui-clients.mdx#get-connection-information) for
more information.)

> Create a connection to a database engine with the proxy endpoint `host`,`port`

Reviewer 3: Same comment as the similar paragraph for Azure Data Studio
